mlir/docs/Tutorials/transform/Ch3.md

```diff
@@ -139,7 +139,20 @@ void MyExtension::init() {
 ```
 
 This type is now directly available in the Transform dialect and can be used in operations.
+In the previous tablegen definition, the type of `$call` must be `Transform_ConcreteOp<“func.call”>`,
+now we make it can to use `CallOpInterfaceHandle`.
 
+```tablegen
+def ChangeCallTargetOp : ... {
+    let arguments = (ins
+    // Specify the type constraint on the input accepting only `func.call` payload
```

Comment:
If I am understanding correctly, the point is to allow `CallOpInterface` ops as well, right? If so, please update the comment.

Reply:
Nevermind. 

Your `fix nit.` is a fine fix.